test/unit/iframe.js

```diff
@@ -1,94 +1,271 @@
+      assert.equal(iframe.googleFonts, constructorConfig.googleFonts);
+      assert.equal(iframe.name, constructorConfig.name);
+    });
+
+    it('sets customStylesHash to an empty object if config does not have customStyles', () => {
+      constructorConfig.customStyles = null;
+      iframe = new Iframe(parent, constructorConfig);
+      assert.deepEqual(iframe.customStylesHash, {});
+    });
+
+    it('sets googleFonts to an empty array if config does not have googleFonts', () => {
+      constructorConfig.googleFonts = null;
+      iframe = new Iframe(parent, constructorConfig);
+      assert.deepEqual(iframe.googleFonts, []);
+    });
+
+    it('does not call setWidth() if there is no width in the config', () => {
+      constructorConfig.width = null;
+      iframe = new Iframe(parent, constructorConfig);
+      assert.notCalled(setWidthStub);
+    });
+
+    it('calls setWidth() if width is in the config', () => {
+      iframe = new Iframe(parent, constructorConfig);
+      assert.calledOnce(setWidthStub);
+      assert.calledWith(setWidthStub, constructorConfig.width);
+    });
+
+    it('sets width to 100%, overflow to hidden, and border to none in iframe\'s style', () => {
+      iframe = new Iframe(parent, constructorConfig);
+      assert.equal(iframe.el.style.width, '100%');
+      assert.equal(iframe.el.style.overflow, 'hidden');
+      assert.include(iframe.el.style.border, 'none');
+    });
+
+    it('disables scrolling, allows transparency, and removes border in iframe\'s attributes', () => {
+      iframe = new Iframe(parent, constructorConfig);
+      assert.equal(iframe.el.getAttribute('horizontalscrolling'), 'no');
+      assert.equal(iframe.el.getAttribute('verticalscrolling'), 'no');
+      assert.equal(iframe.el.getAttribute('allowTransparency'), 'true');
+      assert.equal(iframe.el.getAttribute('frameBorder'), '0');
+      assert.equal(iframe.el.getAttribute('scrolling'), 'no');
     });
 
-    it('appends an iframe', () => {
-      assert.equal('IFRAME', parent.children[0].tagName);
+    it('sets element name to name in config', () => {
+      assert.equal(iframe.el.getAttribute('name'), constructorConfig.name);
     });
 
-    it('appends style tag to head', () => {
-      const styleTags = iframe.document.head.getElementsByTagName('style');
-      assert.equal(styleTags.length, 1);
+    it('sets styleTag to null', () => {
+      iframe = new Iframe(parent, constructorConfig);
+      assert.equal(iframe.styleTag = null);
     });
   });
 
-  describe('loadFontScript', () => {
-    let scriptTags;
+  describe('prototype methods', () => {
+    let parent;
+    let iframe;
+
     beforeEach(() => {
-      scriptTags = document.head.getElementsByTagName('script').length;
-      iframe.googleFonts = ['Lato'];
-      return iframe.load();
+      parent = document.createElement('div');
+      parent.setAttribute('id', 'fixture');
+      document.body.appendChild(parent);
```

Comment:
Could these be moved into a global `beforeEach`, since they're in the repeated in the constructor and prototype methods describe blocks?